src/CurrencyFormat.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Adsmurai\Currency;
+
+use Adsmurai\Currency\Contracts\CurrencyFormat as CurrencyFormatInterface;
+
+class CurrencyFormat implements CurrencyFormatInterface
+{
+    /**
+     * @var string
+     */
+    private $decimalsSeparator;
+    /**
+     * @var string
+     */
+    private $thousandsSeparator;
+    /**
+     * @var int
+     */
+    private $extraPrecision;
+    /**
+     * @var int
+     */
+    private $decorationType;
+    /**
+     * @var int
+     */
+    private $precision;
+
+    public function __construct(
+        string $decimalsSeparator = '.',
+        string $thousandsSeparator = '',
+        int $extraPrecision = 0,
+        int $decorationType = self::DECORATION_NO_DECORATION,
```

Comment:
This default does not match the current behavior (see the last three lines of the `Currency::format` method)